autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/ui/credential/management/AutofillSettingsViewModel.kt

```diff
@@ -161,6 +161,14 @@ class AutofillSettingsViewModel @Inject constructor(
         addCommand(ShowListMode)
     }
 
+    fun allowSaveInEditMode(saveable: Boolean) {
+        if (_viewState.value.credentialMode is Editing) {
```

Comment:
Nit: should we introduce a variable for `viewState.value` to remove the 3x repetitions